lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -73,6 +70,52 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * List the import conditions found in the prelude of an `@import` rule
+ *
+ * @param {Node[]} params
+ * @typedef {import('postcss-value-parser').Node} Node
+ * @returns {Array<string>}
+ */
+function listImportConditions(params) {
+	if (!params.length) return [];
+
+	/** @type {Array<Array<Node>>} */
+	let media = [];
+	/** @type {Array<Node>} */
+	let lastMediaQuery = [];
+
+	for (let i = 0; i < params.length; i++) {
+		const param = params[i];
+
+		if (!param) break;
+
+		// remove whitespace to get a consistent key
+		if (param.type === 'space') {
+			continue;
+		}
+
+		// remove comments to get a consistent key
+		if (param.type === 'comment') {
+			continue;
+		}
+
+		if (param.type === 'div' && param.value === ',') {
+			media.push(lastMediaQuery);
+			lastMediaQuery = [];
+			continue;
+		}
+
+		lastMediaQuery.push(param);
+	}
+
+	if (lastMediaQuery.length) {
+		media.push(lastMediaQuery);
+	}
+
+	return media.map((m) => valueParser.stringify(m).trim()).filter((n) => n.length);
```

Comment:
[suggest] Since `space` nodes are filtered out of `media` nodes, we may simplify the code: 🤔 

```suggestion
	return media.map((m) => valueParser.stringify(m));
```

Although keeping it as-is has no problem.

Reply:
good call!